### PR TITLE
SRCH-5271 updated aria labels for navs

### PR DESCRIPTION
**Pull request by yogesh27**
## Summary
- updated aria labels for navs

### Checklist
Please ensure you have addressed all concerns below before marking a PR "ready for review" or before requesting a re-review. If you cannot complete an item below, replace the checkbox with the ⚠️ `:warning:` emoji and explain why the step was not completed.
 
#### Functionality Checks
 
- [x] You have merged the latest changes from the target branch (usually `main`) into your branch.
  
- [x] Your primary commit message is of the format **SRCH-#### \<description\>** matching the associated Jira ticket.

- [x] PR title is either of the format **SRCH-#### \<description\>** matching the associated Jira ticket (i.e. "SRCH-123 implement feature X"), or **Release - SRCH-####, SRCH-####, SRCH-####** matching the Jira ticket numbers in the release.
 
- [x] Automated checks pass. If Code Climate checks do not pass, explain reason for failures:
 
#### Process Checks

- [x] You have specified at least one "Reviewer".

### CODE REVIEW

File: app/javascript/components/Header/ExtendedHeader.tsx [hunk 1]
```diff
@@ -83,6 +83,7 @@ export const ExtendedHeader = ({ page, toggleMobileNav, mobileNavOpen, primaryHe
         {showMobileMenu && <NavMenuButton onClick={toggleMobileNav} label="Menu" />}
       </div>
       <ExtendedNav
+        aria-label="Primary navigation"
```
Comment (stevenbarragan) [inline app/javascript/components/Header/ExtendedHeader.tsx:86]: Do we need to translate this?

**Review by stevenbarragan — CHANGES_REQUESTED** [review]

File: app/javascript/components/Header/ExtendedHeader.tsx (hunk 1 above)
Comment (yogesh27) [reply app/javascript/components/Header/ExtendedHeader.tsx:86]: @stevenbarragan - good point.

@jimkreft Can you confirm if we need translations for below aria-labels we are using?

1. `aria-label="Primary navigation"`
2. `aria-label="Vertical navigation"`

Comment (jimkreft) [reply app/javascript/components/Header/ExtendedHeader.tsx:86]: @yogesh27 yes, if it's prose, it should be pulling from the localizations YAML.

Comment (yogesh27) [reply app/javascript/components/Header/ExtendedHeader.tsx:86]: @jimkreft One suggestion I need, I am currently using "Vertical navigation" as an aria-label for our vertical navigation links. Is this naming fine? or we can have "Secondary navigation"?

Comment (jimkreft) [reply app/javascript/components/Header/ExtendedHeader.tsx:86]: I think secondary navigation works better

**Review by stevenbarragan — APPROVED** [review]

**Pull request merged**